store/CSDK/CarbonReader.cpp

```diff
@@ -74,27 +75,41 @@ jobject CarbonReader::withHadoopConf(char *key, char *value) {
     return carbonReaderBuilderObject;
 }
 
+jobject CarbonReader::withBatch(int batch) {
+    jclass carbonReaderBuilderClass = jniEnv->GetObjectClass(carbonReaderBuilderObject);
+    jmethodID buildID = jniEnv->GetMethodID(carbonReaderBuilderClass, "withBatch",
```

Comment:
Need to add validation. GetObjectClass, GetMethodID can return null.

Also need to take java exception to cpp for CallObjectMethodA.

Reply:
GetObjectClass shouldn't null, we should check whether carbonReaderBuilderObject is null.
checked GetMethodID.
CallObjectMethodA of withBatch won't throw exception